### PR TITLE
Test the full expected output value.

### PR DESCRIPTION
**Pull request by adelton**
## Description
Test the full expected output value. For the given `data.json` input, the model should give deterministic output and we already have the value in `output.json`.

## How Has This Been Tested?

Manually, the
```
$ oc apply -k tekton/aiedge-e2e/
$ oc create -f tekton/aiedge-e2e/aiedge-e2e.tensorflow-housing.pipelinerun.yaml
$ oc create -f tekton/aiedge-e2e/aiedge-e2e.bike-rentals.pipelinerun.yaml
```
steps pass.

## Merge criteria:

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: pipelines/tekton/aiedge-e2e/tasks/test-model-rest-svc-task.yaml [hunk 1]
```diff
@@ -31,8 +31,7 @@ spec:
         # Check response:
         echo "Check response:"
         cat /tmp/output.json
-        # https://stackoverflow.com/a/7167115/19020549
-        grep -Pazo '\"predictions\":\s*\[\s*[-+]?([0-9]*\.[0-9]+|[0-9]+)*\s*\]' /tmp/output.json
+        test "$( python3 -m json.tool --no-indent < $(workspaces.test-data.path)/output.json )" = "$( python3 -m json.tool --no-indent < /tmp/output.json )"
```
Comment (MarianMacik) [inline pipelines/tekton/aiedge-e2e/tasks/test-model-rest-svc-task.yaml:34]: This also works
```suggestion
        test "$( python3 -m json.tool --no-indent $(workspaces.test-data.path)/output.json )" = "$( python3 -m json.tool --no-indent /tmp/output.json )"
```
and it allows for a nicer log, explicitly showing from which files the content was read. Currently the log looks like this (from the PR check):
```bash
++ python3 -m json.tool --no-indent
++ python3 -m json.tool --no-indent
+ test '{"predictions": [-28.7639923]}' = '{"predictions": [-28.7639923]}'
```

```bash
++ python3 -m json.tool --no-indent ${some-workspace-path}/output.json
++ python3 -m json.tool --no-indent /tmp/output.json
+ test '{"predictions": [-28.7639923]}' = '{"predictions": [-28.7639923]}'
```
I guess the second example is easier for possible debugging, what do you think?

**Review by MarianMacik — COMMENTED** [review]
Thanks for looking into this! Just one small suggestion.

File: pipelines/tekton/aiedge-e2e/tasks/test-model-rest-svc-task.yaml (hunk 1 above)
Comment (adelton) [reply pipelines/tekton/aiedge-e2e/tasks/test-model-rest-svc-task.yaml]: Good point, updated with the input files as parameters.

**Review by MarianMacik — APPROVED** [review]
/lgtm

**Review by grdryn — APPROVED** [review]
/approve

**Pull request merged**